run.py

```diff
@@ -103,12 +103,19 @@ def main(args):
                 )
 
                 feature_names = model.get_human_readable_feature_names()
-                for i, (importance, index, is_positive) in enumerate(
-                    importance["importances"]
-                ):
-                    print(
-                        f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
-                    )
+
+                # shap_values are stored in class 0 for binary classification
```

Comment:
I don't like this part, users of the `print_feature_importance` function should not have to know about this. We should move it to print_feature_importances (making `probas` the parameter of print_feature_importances instead of `predicted_class`)